### DISCUSSION TITLE
[Feature] customizable toggle for sorting methods

### DISCUSSION THREAD

**Discussion by WhiteCranes** [immich-app/immich#14337] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When I use External Libraries, the timeline is generated in the frontend(app&web) end based on the EXIF information of the media files. However, the Create Date in the EXIF information of the media files I added to External Libraries is not accurate, or I do not care about it. I need to sort the files in the front end based on the time I added them. I hope to add a customizable toggle for sorting methods.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Dupe of #4170. You should fix your metadata though.
